src/main/java/org/primefaces/util/CalendarUtils.java

```diff
@@ -138,19 +138,19 @@ public static final String getTimeOnlyValueAsString(FacesContext context, UICale
 
     public static final String getValueAsString(FacesContext context, UICalendar calendar, Object value, String pattern) {
         if (value instanceof List) {
-            String valuesAsString = "";
+            StringBuilder valuesAsString = new StringBuilder("");
```

Comment:
Use Constants.EMPTY_STRING instead of "".

Reply:
IMO, you should keep it simple: `StringBuilder valuesAsString = new StringBuilder();` In both case, initial capacity is the same